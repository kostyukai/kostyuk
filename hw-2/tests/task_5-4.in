12 1 13 2 7 
1.5 2.6 7.4 1.2 7.3
