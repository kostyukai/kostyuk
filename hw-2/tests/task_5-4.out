35
20.00
